### PR TITLE
[Application] Add examples for launchAppControl and findAppControl

### PR DESCRIPTION
**Pull request by tiwanek**
BUG=XWALK-3671

### CODE REVIEW

**Comment by rakuco** [comment]
yay tizen. Can you resubmit so that the package upgrades finish?

File: examples/application.html [hunk 1]
```diff
@@ -187,6 +211,89 @@
...
+    var data = [
+      new tizen.ApplicationControlData("one", ["first", "second"]),
+      new tizen.ApplicationControlData("zwei", ["erster", "zweiter"]),
+    ];
+    var appControl = new tizen.ApplicationControl(
+                          operation.value ? operation.value : null,
+                          uri.value ? uri.value : null,
+                          mime.value ? mime.value : null,
+                          category.value ? category.value : null,
+                          data);
```
Comment (halton) [inline examples/application.html]: indent

File: examples/application.html [hunk 2]
```diff
@@ -187,6 +211,89 @@
...
+    var uri = document.getElementById("app_find_app_control_uri");
+    var mime = document.getElementById("app_find_app_control_mime");
+    var category = document.getElementById("app_find_app_control_category");
+    var output = document.getElementById("app_find_app_control");
+
+    var appControl = new tizen.ApplicationControl(
+                          operation.value ? operation.value : null,
+                          uri.value ? uri.value : null,
+                          mime.value ? mime.value : null,
+                          category.value ? category.value : null);
```
Comment (halton) [inline examples/application.html]: indent

**Comment by halton** [comment]
LGTM with the two nit indent comments.

**Pull request merged**